src/layout/PersonLookup/PersonLookupComponent.tsx

```diff
@@ -132,11 +134,15 @@ export function PersonLookupComponent({ node, overrideDisplay }: PropsFromGeneri
 
     const { data } = await performLookup();
     if (data?.person) {
-      setValue('person_lookup_name', data.person.name);
+      setValue('person_lookup_name', getFullName(data.person));
       setValue('person_lookup_ssn', data.person.ssn);
     }
   }
 
+  function getFullName({ firstName, middleName, lastName }) {
+    return middleName ? `${firstName} ${middleName} ${lastName}` : `${firstName} ${lastName}`;
```

Comment:
Trenger ikke endre på det nå, bare et tips i tilfelle det kan bli handy en gang du har et litt mer komplekst case enn her og du ikke har sett det før 😄  
Jeg syns det ofte er nice å legge sammen tekst vha en liste. Typ
`[firstName, middleName, lastName].filter(it != null).join(" ")`